main/nav_backplane_viewer.py

```
@@ -0,0 +1,1366 @@
+                    rgba = im.convert('RGBA')
+                    self._summary_rgba = np.array(rgba)
+                if self._summary_rgba is not None:
+                    self._logger.info('Loaded summary overlay "%s" size=%dx%d',
+                                      png_local,
+                                      self._summary_rgba.shape[1], self._summary_rgba.shape[0])
+                self._summary_enable.setEnabled(True)
+            else:
+                self._summary_rgba = None
+                self._summary_enable.setChecked(False)
+                self._summary_enable.setEnabled(False)
+        except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)